bin/options.js

```diff
@@ -3,23 +3,58 @@
 /**
  * Dependencies.
  */
-
 const fs = require('fs');
+const path = require('path');
+const readPkgUp = require('read-pkg-up');
 
 /**
  * Export `getOptions`.
+ * Export `getTestDirectory`.
  */
+module.exports = {
+  getOptions: getOptions,
+  getTestDirectory: getTestDirectory
+};
 
-module.exports = getOptions;
+/**
+ * Default test directory.
+ *
+ * @constant
+ * @type {string}
+ * @default
+ */
+const DEFAULT_TEST_DIRECTORY = 'test';
 
 /**
- * Default pathname for run-control file.
+ * Default filename of run-control file.
  *
  * @constant
  * @type {string}
  * @default
  */
-const defaultPathname = 'test/mocha.opts';
+const DEFAULT_OPTS_FILENAME = 'mocha.opts';
+
+/**
+ * Reads directories.test of package.json.
+ *
+ * @public
+ * @returns {string} test directory
+ */
+let cachedTestDirectory;
+
+function getTestDirectory() {
+  if (!cachedTestDirectory) {
```

Comment:
know what though?  if you use `require('/path/to/some/package.json')`, Node.js *will cache it for you.*  that's another reason why `findup-sync` makes more sense here, IMO--it just gives you a path.